README.md

```diff
@@ -1,7 +1,7 @@
 <!-- markdownlint-disable-next-line -->
 **This repo is *not* included in the Hacktoberfest event, as it is for practice only!**
 
-We have other repositories in the organisation you can contribute to. If you would like to join our GitHub organisation, raise an issue on this repo [EddieHubCommunity-Support](https://github.com/EddieHubCommunity/support)
+We have other repositories in the organization you can contribute to. If you would like to join our GitHub organization, raise an issue on this repo [EddieHubCommunity-Support](https://github.com/EddieHubCommunity/support)
```

Comment:
This is UK English, so maybe we should keep it as-is. 🙂

Reply:
```suggestion
We have other repositories in the organisation you can contribute to. If you would like to join our GitHub organisation, raise an issue on this repo [EddieHubCommunity-Support](https://github.com/EddieHubCommunity/support)
```